src/meta-assistant/slack/slack-utils.ts

```diff
@@ -53,49 +53,44 @@ namespace SlackUtilities {
   /**
    * Create message based on specific users timebank data
    *
-   * @param user timebank data
+   * @param user severa data
    * @param numberOfToday Todays number
    * @returns string message if id match
    */
   const constructDailyMessage = (user: DailyCombinedData, numberOfToday: number): DailyMessageData => {
-    const { name, date, firstName, minimumBillableRate } = user;
-
-    const displayDate = DateTime.fromISO(date).toFormat("dd.MM.yyyy");
-
-    const {
-      logged,
-      loggedProject,
-      expected,
-      internal,
-      billableProject,
-      nonBillableProject
+    const { firstName, date, minimumBillableRate } = user;
+
+    const { 
+      enteredHours, 
+      expectedHours, 
+      quantity,
+      totalBillableTime,
+      nonBillableProject,
     } = TimeUtilities.handleTimeFormatting(user);
 
     const {
       message,
       billableHoursPercentage
     } = MessageUtilities.calculateWorkedTimeAndBillableHours(user);
 
+    const displayDate = DateTime.fromISO(date).toFormat("dd.MM.yyyy");
+  
     const customMessage = `
-Hi ${firstName},
-${numberOfToday === 1 ? "Last friday" :"Yesterday"} (${displayDate}) you worked ${logged} with an expected time of ${expected}.
-${message}
-Logged project time: ${loggedProject}, Billable project time: ${billableProject}, Non billable project time: ${nonBillableProject}, Internal time: ${internal}.
-Your percentage of billable hours was: ${billableHoursPercentage}% ${parseInt(billableHoursPercentage) >= minimumBillableRate ? ":+1:" : ":-1:"}
-Have a great rest of the day!
-    `;
-
+      Hi ${firstName},
+      ${numberOfToday === 1 ? "Last friday" :"Yesterday"} (${displayDate}) you worked ${enteredHours} with an expected time of ${expectedHours}.
+      ${message}
+      Logged project time: ${quantity}, Billable project time: ${totalBillableTime}, Non billable project time: ${nonBillableProject}.
+      Your percentage of billable hours was: ${billableHoursPercentage}% ${parseInt(billableHoursPercentage) >= minimumBillableRate ? ":+1:" : ":-1:"}
```

Comment:
```suggestion
      Your percentage of billable hours was: ${billableHoursPercentage}% ${Number.parseInt(billableHoursPercentage) >= minimumBillableRate ? ":+1:" : ":-1:"}
```

Reply:
My ide is complaining at this line , it suggests to change to Number.parseInt, i dont know do you have it on urs , if no then no need to change